ui/webui/src/users/UserAccount.jsx

```diff
@@ -15,16 +15,65 @@
  * along with This program; If not, see <http://www.gnu.org/licenses/>.
  */
 import cockpit from 'cockpit';
-import React from 'react';
+import React, { useState, useContext, /* useEffect */ } from 'react';
 
 import {
+    Form,
+    FormGroup,
+    TextInput,
     PageSection
 } from '@patternfly/react-core';
 
-import { Header } from '../Common.jsx';
+import { useEvent, useObject } from 'hooks';
+
+import { AddressContext, Header } from '../Common.jsx';
 
 export const UserAccount = () => {
+    const [name, setName] = useState();
+    const [gecos, setGecos] = useState();
+    const address = useContext(AddressContext);
+
+    const usersProxy = useObject(() => {
+        const client = cockpit.dbus('org.fedoraproject.Anaconda.Modules.Users', { superuser: 'try', bus: 'none', address });
+        const proxy = client.proxy(
+            'org.fedoraproject.Anaconda.Modules.Users',
+            '/org/fedoraproject/Anaconda/Modules/Users',
+        );
+
+        return proxy;
+    }, null, [address]);
+
+    const onUsernameChange = (value) => {
+        // console.log('onUsernameChange: ' + value);
+        setName(value);
+    };
+
+    const onGecosChange = (value) => {
+        // console.log('onGecosChange: ' + value);
+        setGecos(value);
+    };
+
+    useEvent(usersProxy, 'changed', (event, data) => {
```

Comment:
The contents of this are called in the following times:
* when the proxy is ready and it's data are ready for reading
* every time some property changes

You are mostly interested in the first bullet, so when the proxy data become available for the first time.

Then you want to initialize the UI with possibly existing users (set from kickstart)

For that it's probably enough to do here:

``` js
if (data.Users.length > 0)
    setName(data.Users[0]).name.v;
```

Similarly for the real name.